docs/deployed-contracts/hoodi.md

```diff
@@ -0,0 +1,118 @@
+- Gate Seal Factory: [`0xA402349F560D45310D301E92B1AA4DeCABe147B3`](https://hoodi.cloud.blockscout.com/address/0xA402349F560D45310D301E92B1AA4DeCABe147B3)
+- Gate Seal: [`0x2168Ea6D948Ab49c3D34c667A7e02F92369F3A9C`](https://hoodi.cloud.blockscout.com/address/0x2168Ea6D948Ab49c3D34c667A7e02F92369F3A9C)
+
+## Data Bus
+
+- DataBus on Chiado (Testnet): [`0x37De961D6bb5865867aDd416be07189D2Dd960e6`](https://gnosis-chiado.blockscout.com/address/0x37De961D6bb5865867aDd416be07189D2Dd960e6)
+
+## Staking modules
+
+### Curated Module
+
+- Node Operators Registry: [`0x5cDbE1590c083b5A2A64427fAA63A7cfDB91FbB5`](https://hoodi.cloud.blockscout.com/address/0x5cDbE1590c083b5A2A64427fAA63A7cfDB91FbB5)
+
+### Simple DVT Module
+
+- Node Operators Registry: [`0x0B5236BECA68004DB89434462DfC3BB074d2c830`](https://hoodi.cloud.blockscout.com/address/0x0B5236BECA68004DB89434462DfC3BB074d2c830)
+
+### Sandbox Module
+
+- Node Operators Registry: [`0x682E94d2630846a503BDeE8b6810DF71C9806891`](https://hoodi.cloud.blockscout.com/address/0x682E94d2630846a503BDeE8b6810DF71C9806891)
+
+### Community Staking Module
+
+- CSModule: [`0x79CEf36D84743222f37765204Bec41E92a93E59d`](https://hoodi.cloud.blockscout.com/address/0x79CEf36D84743222f37765204Bec41E92a93E59d) (proxy)
+- CSAccounting: [`0xA54b90BA34C5f326BC1485054080994e38FB4C60`](https://hoodi.cloud.blockscout.com/address/0xA54b90BA34C5f326BC1485054080994e38FB4C60) (proxy)
+- CSFeeDistributor: [`0xaCd9820b0A2229a82dc1A0770307ce5522FF3582`](https://hoodi.cloud.blockscout.com/address/0xaCd9820b0A2229a82dc1A0770307ce5522FF3582) (proxy)
+- CSVerifier: [`0x16D0f6068D211608e3703323314aa976a6492D09`](https://hoodi.cloud.blockscout.com/address/0x16D0f6068D211608e3703323314aa976a6492D09)
+- CSEarlyAdoption: [`0x3281b9E45518F462E594697f8fba1896a8B43939`](https://hoodi.cloud.blockscout.com/address/0x3281b9E45518F462E594697f8fba1896a8B43939)
+- GateSeal: [`0xEe1f7f0ebB5900F348f2CfbcC641FB1681359B8a`](https://hoodi.cloud.blockscout.com/address/0xEe1f7f0ebB5900F348f2CfbcC641FB1681359B8a)
+- CSFeeOracle:
+  - CSFeeOracle: [`0xe7314f561B2e72f9543F1004e741bab6Fc51028B`](https://hoodi.cloud.blockscout.com/address/0xe7314f561B2e72f9543F1004e741bab6Fc51028B) (proxy)
+  - HashConsensus: [`0x54f74a10e4397dDeF85C4854d9dfcA129D72C637`](https://hoodi.cloud.blockscout.com/address/0x54f74a10e4397dDeF85C4854d9dfcA129D72C637)
+- External libraries:
+  - AssetRecovererLib: [`0xa0513a7e28dac4c31b5ccbf9a5f474b759257985`](https://hoodi.cloud.blockscout.com/address/0xa0513a7e28dac4c31b5ccbf9a5f474b759257985)
+  - NOAddresses: [`0x479244bac2ae1d64841753307a0552183642c121`](https://hoodi.cloud.blockscout.com/address/0x479244bac2ae1d64841753307a0552183642c121)
+  - QueueLib: [`0x6fb7af5addb044182caa27db35e394ed3451a4da`](https://hoodi.cloud.blockscout.com/address/0x6fb7af5addb044182caa27db35e394ed3451a4da)
```

Comment:
aren't mentioned on the deployed artifacts

Reply:
They could be found in this artifact: https://github.com/lidofinance/community-staking-module/blob/865759f5d86798a7b6ea839cb26f313858114601/artifacts/hoodi/transactions.json#L2844-L2847

Reply:
gotcha!